cli/azd/pkg/project/importer.go

```diff
@@ -130,7 +146,32 @@ func (im *ImportManager) ProjectInfrastructure(ctx context.Context, projectConfi
 	}
 
 	return nil, fmt.Errorf(
-		"this project does not contain any infrastructure, have you created an '%s' folder?", filepath.Base(infraRoot))
+		"this project does not contain expected infrastructure, folder: '%s' and module: '%s'",
+		infraRoot,
+		projectConfig.Infra.Module)
+}
+
+func pathHasModule(path, module string) (bool, error) {
+	var moduleFound bool
+	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
```

Comment:
Do we intend to do this recursively -- i.e., should `infra/sub/main.bicep` be considered? `filepath.Walk` will currently do this.

Alternatively, we can probably instead perform `os.ReadDir` and do the same comparison.